### ISSUE TITLE
[Feature] Project "Status update"

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#6995]
"Status updates" will provide users with ~1-4 sentences on the current state of the project. This is the content people receive by email/text/etc. if they follow the project. Read together, these will provide a project history. 

### Comments page (edit)
Add comment types and ability to filter comments by type #7124 

### Summary page (display)
Render the most recent "Status update" comment on the "Summary" page. 
- If the user clicks the pencil icon, they edit the existing status update comment. 
- If the user clicks the [Add Circle Outline](https://fonts.google.com/icons?selected=Material%20Icons%20Outlined%3Aadd_circle_outline%3A), they get a blank field
 
![New Summary Page – 3 0](https://user-images.githubusercontent.com/1463708/134239231-f800be2f-4368-415c-8fe9-7a7d91beb2e5.png)

Edit existing status update:
![New Summary Page – 3 0 – Edit status](https://user-images.githubusercontent.com/1463708/134239229-9c537c11-d569-44db-b01b-78243033779a.png)

Create new status update: 
![New Summary Page – 3 0 – New status](https://user-images.githubusercontent.com/1463708/134253895-c2c9875c-13e2-4504-b84b-4a44d85b9956.png)

**Comment by amenity** [comment]
Actually, this might also work for #6409 - if we had a "Note" field in the table and we could access the past notes.

**Comment by amenity** [comment]
To prevent tons of noise from typo fixes, could have a "new" status or an "edit" status.

**Comment by johnclary** [comment]
@amenity is this good to go?

**Comment by amenity** [comment]
Yep!

**Comment by sergiogcx** [comment]
A PR is now available and ready to be deployed in moped-test. Sorry it took this long, I kept getting distracted. Also, this was a bit more complicated than I thought. 

(Be sure to deploy to moped-test, just in case)
https://moped-test.austinmobility.io/moped/projects/156/

I have questions about the UI decisions I made, I am not fixated in the behavior or the looks. The code should be able to handle most changes to the UI in terms of components.
